NEWS.md

```diff
@@ -6,52 +6,245 @@ bug fixes (and other actions) for each version of Libfabric since
 version 1.0.  New major releases include all fixes from minor
 releases with earlier release dates.
 
-v1.17.0, Fri Dec 9, 2022
+v1.18.0, Fri Apr 7, 2023
 ========================
 
 ## Core
 
-- TODO
+## EFA
+
+## Hooks
+
+## Net
+
+## OPX
+
+## PSM3
+
+## RXM
+
+## SHM
+
+## TCP
+
+## UCX
+
+## Util
+
+## Verbs
+
+## Fabtests
+
+
+v1.17.1, Fri Mar 3, 2023
+========================
+
+## Core
+
+- Fix spinlocks for macOS
+- hmem_cuda Add const to param to remove warning
+- Fix typos in fi_ext.h
+- ofi_epoll: Remove unused hot_index struct member
 
 ## EFA
```

Comment:
Why are there two EFA sections? Other is on line 123

Reply:
These updates are copied from the readme file in the v1.17.x branch.  One contains the updates for the v1.17.0 release, the other v1.17.1.  The updates for the v1.18.0 release is empty and at the top of the file.